test/performance/alignment/test_sequence_generation.hpp

```diff
@@ -0,0 +1,57 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <seqan3/test/performance/sequence_generator.hpp>
+
+inline constexpr size_t default_sequence_length = 150;
+inline constexpr size_t default_sequence_length_variance = 0;
+inline constexpr size_t default_set_size = 500;
+
+template <typename alphabet_t>
+auto generate_data_seqan3(size_t sequence_length = default_sequence_length,
+                          size_t sequence_length_variance = default_sequence_length_variance,
+                          size_t set_size = default_set_size)
+{
+    using sequence_t = decltype(seqan3::test::generate_sequence<alphabet_t>());
+
+    std::vector<sequence_t> vec1;
+    std::vector<sequence_t> vec2;
+
+    for (unsigned i = 0; i < set_size; ++i)
+    {
+        vec1.push_back(seqan3::test::generate_sequence<alphabet_t>(sequence_length,
+                                                                   sequence_length_variance,
+                                                                   i));
+        vec2.push_back(seqan3::test::generate_sequence<alphabet_t>(sequence_length,
+                                                                   sequence_length_variance,
+                                                                   i + set_size));
+    }
+    return std::pair{vec1, vec2};
```

Comment:
Why don't you just extend the existing `generate_sequence_pairs` function to support an optional deviation parameter? This is almost a copy of that function.

Reply:
> Why don't you just extend the existing generate_sequence_pairs function to support an optional deviation parameter? This is almost a copy of that function.

Well spotted. I just copied from other benchmarks, which seemed to be older than changes made to the sequence generator file.

Reply:
not resolved.
Even if you do need new functions they would love in `/test/include`

Reply:
yes, right. 
... and make a 👶 there 😄